.github/workflows/release.yml

```diff
@@ -50,3 +50,21 @@ jobs:
           files: |
             pie.phar
             pie.phar.asc
+      - name: Get release tag
+        run: |
+          RELEASE_TAG=${GITHUB_REF/refs\/tags\//}
+          echo "RELEASE_TAG=$RELEASE_TAG" >> $GITHUB_ENV
+      - name: Update Homebrew formula
+        uses: mislav/bump-homebrew-formula-action@v3
```

Comment:
I don't really understand what this is for / what this is doing? I'm assuming something to do with brew, but it's something I'm unfamiliar with. Can you give me a TLDR? or point me to some docs to understand this process please?